submissions/Harnytskyi/js-pre-oop/index.js

```diff
@@ -0,0 +1,77 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Harnytskyi/a-tiny-JS-world
+   Web app: https://harnytskyi.github.io/a-tiny-JS-world 
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+    species: 'human',
+    name: 'John',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    saying: 'Hello, everybody!'
+};
+
+const woman = {
+    species: 'human',
+    name: 'Jessica',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: 'Good morning, gays!'
+}
+
+const cat = {
+    species: 'cat',
+    name: 'Tom',
+    gender: 'male',
+    legs: 4,
+    saying: 'Meow!'
+};
+
+const dog = {
+    species: 'dog',
+    name: 'Jerry',
+    gender: 'female',
+    legs: 4,
+    saying: 'Gav!'
+};
+
+const inhabitants = [man, woman, cat, dog];
+
+function message(obj) {
+    const infoMessage = [
```

Comment:
The array here represents not a message but rather components of a message yet to build.